--- CRISPRCasFinder.pl
+++ CRISPRCasFinder.pl
@@ -33,12 +33,19 @@
 
 # TODO add URLescape module in case the GFF attributes get messy
 #local modules
-my $version = "4.2.20";
+my $version = "4.2.30";
 
 # set parameters for the Vmatch program
 #$ENV{'LD_LIBRARY_PATH'} = '.';
 
 ## Parameters
+
+my $PathVmatch2 = "/usr/local/CRISPRCasFinder/bin/vmatch2";
+my $PathMkvtree2 = "/usr/local/CRISPRCasFinder/bin/mkvtree2";
+my $PathVsubseqselect2 = "/usr/local/CRISPRCasFinder/bin/vsubseqselect2";
+my $PathMacsyfinder = "/usr/local/CRISPRCasFinder/macsyfinder";
+
+
 my $SpSim = 60; # maximal allowed percentage of similarity between Spacers (default value=60)
 
 my $Sp2 = 2.5; # maximal Spacers size in function of DR size (default value=2.5)
@@ -63,7 +70,7 @@
 
 my $launchCasFinder = 0; # boolean variable indicating if we use casfinder or not (default value=0)
 
-my $casfinder = "CasFinder-2.0.3"; # repository containing new CasFinder (default 'CasFinder-2.0.3')
+my $casfinder = "CasFinder-2.0.4"; # repository containing new CasFinder (default 'CasFinder-2.0.4')
 
 my $kingdom = "Bacteria"; # allow to choose analysis between Archaea and Bacteria (default 'Bacteria')
 
@@ -588,7 +595,7 @@
   	}
   	#Modification DC - 05/05/2017
   	#makesystemcall("./vmatch " . join(' ',@vmatchoptions)); #DC
-  	makesystemcall("vmatch2 " . join(' ',@vmatchoptions)); #LK - DC replaced vmatch by vmatch2
+  	makesystemcall("$PathVmatch2 " . join(' ',@vmatchoptions)); #LK - DC replaced vmatch by vmatch2
   	#makesystemcall("cp vmatch_result.txt CHECKvmatch_result.txt");
   
   	my @rep = trans_data("vmatch_result.txt");
@@ -1366,18 +1373,18 @@
   my $addToMacSy = "";
   #manage $definition info
   if( ($definition eq "General") or ($definition eq "general") or ($definition eq "G") or ($definition eq "g") ){
-	$casdb = $casfinder."/DEF-Class-2.0.3/";  # DEF-General-2.0 replaced by DEF-Class-2.0.2 then replaced by DEF-Class-2.0.3
+	$casdb = $casfinder."/DEF-Class-2.0.4/";  # DEF-General-2.0 replaced by DEF-Class-2.0.2 then replaced by DEF-Class-2.0.4
 	$addToMacSy .= "--min-genes-required General-Class1 1 "; # General-1CAS replaced by General-Class1
 	$addToMacSy .= "--min-genes-required General-Class2 1 "; # General-3CAS replaced by General-Class2
 	#$default = 1;
   }
   elsif( ($definition eq "Typing") or ($definition eq "typing") or ($definition eq "T") or ($definition eq "t") ){
-  	$casdb = $casfinder."/DEF-Typing-2.0.3/"; # DEF-Typing-2.0 replaced by DEF-Typing-2.0.2 then replaced by DEF-Typing-2.0.3
+  	$casdb = $casfinder."/DEF-Typing-2.0.4/"; # DEF-Typing-2.0 replaced by DEF-Typing-2.0.2 then replaced by DEF-Typing-2.0.4
 	#$addToMacSy .= "--min-genes-required CAS 1 --min-genes-required CAS-TypeI 1 --min-genes-required CAS-TypeII 1 --min-genes-required CAS-TypeIII 1 --min-genes-required CAS-TypeIV 1 --min-genes-required CAS-TypeV 1 ";
 	#$addToMacSy .= "--min-genes-required CAS-TypeVI 1 ";
   }
   elsif( ($definition eq "SubTyping") or ($definition eq "subtyping") or ($definition eq "S") or ($definition eq "s") ){
-  	$casdb = $casfinder."/DEF-SubTyping-2.0.3/"; # DEF-SubTyping-2.0 replaced by DEF-SubTyping-2.0.2 then replaced by DEF-SubTyping-2.0.3
+  	$casdb = $casfinder."/DEF-SubTyping-2.0.4/"; # DEF-SubTyping-2.0 replaced by DEF-SubTyping-2.0.2 then replaced by DEF-SubTyping-2.0.4
   }
   else{
   	#$casdb = $casfinder."/DEF-General-2.0/";
@@ -1386,7 +1393,7 @@
   
   $casdb =~ s/\/\//\//; #DC - replace '//' by '/'
 
-  my $profiles = $casfinder."/CASprofiles-2.0.3/"; # CASprofiles-2.0 replaced by CASprofiles-2.0.2 then replaced by CASprofiles-2.0.3
+  my $profiles = $casfinder."/CASprofiles-2.0.4/"; # CASprofiles-2.0 replaced by CASprofiles-2.0.2 then replaced by CASprofiles-2.0.4
   $profiles =~ s/\/\//\//; #DC - replace '//' by '/'
 
   my $results = $ResultDir."/Cas_REPORT.tsv"; # former name: $ResultDir."/Cas_".$RefSeq.".txt";
@@ -1539,7 +1546,7 @@
     my ($macsyfinder_hour,$macsyfinder_min,$macsyfinder_sec) = Now();
     if ( (-d $casfinder) and (-d $casdb) and (-d $profiles) ) {
 	  #if($default){
-	    $macsyfinder = "macsyfinder -w $cpuMacSyFinder -d $casdb -p $profiles --sequence-db $proteome all --out-dir $casDir ";
+	    $macsyfinder = "$PathMacsyfinder -w $cpuMacSyFinder -d $casdb -p $profiles --sequence-db $proteome all --out-dir $casDir ";
 	    #$macsyfinder = "macsyfinder -w $cpuMacSyFinder -d $casdb -p $profiles --sequence-db $proteome all --out-dir $casDir "; 
 	    $macsyfinder .= $addToMacSy; # --multi-loci General-CAS
 	  #}
@@ -1551,7 +1558,7 @@
         #$macsyfinder = "macsyfinder -w $cpuMacSyFinder --sequence-db $proteome all --out-dir $casDir"; #MacSyfinder without new definitions/profiles
     #}
     else{  
-    	$macsyfinder = "macsyfinder -w $cpuMacSyFinder --sequence-db $proteome all --out-dir $casDir"; #MacSyfinder without new definitions/profiles
+    	$macsyfinder = "$PathMacsyfinder -w $cpuMacSyFinder --sequence-db $proteome all --out-dir $casDir"; #MacSyfinder without new definitions/profiles
     }
     #print "$macsyfinder\n";
     #print "METAGENOME = $metagenome ; ACTUAL VALUE = $actualMetaOptionValue\n\n\n";
@@ -3647,7 +3654,8 @@
 			exit EX_CONFIG;
 		}
  	}	
-     return $found;
+  $found= 1; # PA
+  return $found;
 }
 
 #------------------------------------------------------------------------------
@@ -3993,7 +4001,7 @@
 
 	# Modification DC - 05/05/2017
     	#makesystemcall("./vsubseqselect " . join(' ',@subselectoptions)); #DC
-    	makesystemcall("vsubseqselect2 " . join(' ',@subselectoptions)); #LK - DC replaced vsubseqselect by vsubseqselect2
+    	makesystemcall("$PathVsubseqselect2 " . join(' ',@subselectoptions)); #LK - DC replaced vsubseqselect by vsubseqselect2
 	my ($hour,$min,$sec) = Now();
 	if($logOption){
    		print LOG "[$hour:$min:$sec] vsubseqselect2 @subselectoptions\n"; # DC replaced vsubseqselect by vsubseqselect2
@@ -5162,7 +5170,7 @@
   my ($inputfile,$indexname) = @_;
   if(not (checkdbfile($inputfile,"${indexname}.prj")))
   {
-    makesystemcall("mkvtree2 -db $inputfile " .   # LK - DC replaced mkvtree by mkvtree2
+    makesystemcall("$PathMkvtree2 -db $inputfile " .   # LK - DC replaced mkvtree by mkvtree2
                    "-dna -pl -lcp -suf -tis -ois -bwt -bck -sti1");
     my ($hour,$min,$sec) = Now();
